dataactvalidator/scripts/loadTas.py

```diff
@@ -131,6 +143,25 @@ def existing_id(row, existing):
         return potential_match.account_num
 
 
+def get_parser():
+    """ Generates list of command-line arguments
+
+        Returns:
+            argument parser to be used for commandline
+    """
+    tas_parser = argparse.ArgumentParser(description='Import data from the cars_tas.csv')
+    tas_parser.add_argument('--tas_file', '-f', type=str, default=None, help='Path to specifc cars_tas file to use')
```

Comment:
Do we have to manually enter every file? It might make more sense to use boto to gather the historical files one at a time in a loop until there are no more rows with empty `budget_function_code` columns OR we run out of historical files.

Reply:
Fixed to work with the versions of the file in reverse chronological order.